.gitlab-ci.yml

```
@@ -89,11 +89,15 @@ trigger:mender-dist-packages:
   stage: trigger
   before_script:
     - apk add --no-cache curl
+    - export MENDER_CLIENT_VERSION=$(git tag | egrep -e '^[0-9]+\.[0-9]+\.[0-9]+$' | sort -V -r | head -n1)
+    # This will only be true in the case that we have a new newest client tag
+    - test ${MENDER_CLIENT_VERSION} = ${CI_COMMIT_TAG} && export PUBLISH_LATEST_CLIENT_PACKAGE="true" || true
```

I would quote the variable in case it is empty for some reason
```suggestion
    - test "${MENDER_CLIENT_VERSION}" = ${CI_COMMIT_TAG} && export PUBLISH_LATEST_CLIENT_PACKAGE="true" || true
```

---

How about, just to be sure, we do:
```
test "${MENDER_CLIENT_VERSION}" = "${CI_COMMIT_TAG:-false}"
```
just to be sure?

---

If we quote, the default value "-false" is not needed.

Actually, now that I see it with fresh eyes, we should use `CI_COMMIT_REF_NAME` (same as we pass to the pipeline below) instead of the TAG one. For tags they will be the same, but for any other build the TAG one will be undefined (hence your default value above).

With regards to the test syntax quoting both is the safest, but if we use CI_COMMIT_REF_NAME we are guaranteed to have a value from GitLab.

---

> With regards to the test syntax quoting both is the safest, but if we use CI_COMMIT_REF_NAME we are guaranteed to have a value from GitLab.

Yes, this is much nicer!

---

One more question:
`    - test "${MENDER_CLIENT_VERSION}" = "${CI_COMMIT_REF_NAME}" && export PUBLISH_LATEST_CLIENT_PACKAGE="true" || true`
Is it more correct to actually let the job fail here? i.e., remove the `||true` suffix?
I think so